.github/workflows/ci.yml

```diff
@@ -6,13 +6,16 @@ on:
   pull_request:
     branches: [ master ]
 
+env:
+  ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
+
 jobs:
   build:
     runs-on: ubuntu-latest
     container: integralsw/osa-python:auto-latest
     
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
looks like this is not really necessary for this issue, but ok, it's good to upgrade if possible

Reply:
Exactly